tests/EnumTest.php

```diff
@@ -150,6 +179,9 @@ public function testIsValidKey()
     public function testSearch()
     {
         $this->assertEquals('FOO', EnumFixture::search('foo'));
-        $this->assertNotEquals('FOO', EnumFixture::isValidKey('baz'));
+        /**
+         * @see https://github.com/myclabs/php-enum/issues/9
+         */
+        $this->assertEquals(EnumFixture::PROBLEMATIC_NUMBER, EnumFixture::search(1));
```

Comment:
No, it's not normal. I've just added that so that I could reference it in another bug ticket, since search() method is also not type safe. I'll fix that soon.